.github/workflows/release.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+name: Release Teleporter
+
+on:
+  push:
+    tags:
+      - "v*.*.0"
+
+jobs:
+  build_and_upload_artifacts:
+    name: Build and Upload Teleporter Artifacts
+    runs-on: ubuntu-20.04
+    env:
+      deployment_tx_fn: TeleporterMessenger_DeployerTransaction_${{ github.ref_name }}.txt
```

Comment:
Nit: Sorry I know this was a pre-existing file name, but could we change it to `DeploymentTransaction` rather than `DeployerTransaction`? 

I think that is more clear/descriptive.